lua/lir/devicons.lua

```diff
@@ -28,13 +28,11 @@ function devicons.setup()
   end
   local _, hi_name = Devicons.get_icon(FOLDER_ICON_NAME)
   if hi_name ~= nil and hi_name:match("IconDefault$") then
```

Comment:
Any reason for doing this check?

This will be `nil` if the `setup` function was not called yet for `nvim-web-devicons` and so the folder icon will not be added.

Reply:
I'm sorry. I forgot why I was checking.
So, if you don't need the check, you can delete it.